addons/account/models/account_move.py

```diff
@@ -497,6 +504,14 @@ def onchange_tax_ids_create_aml(self):
         for line in self:
             line.recompute_tax_line = True
 
+    @api.onchange('debit', 'credit')
+    def onchange_debit_credit(self):
+        for line in self:
```

Comment:
this logic is not good :

start: debit : 0, credit :0
write in debit : 10
result : debit : 10, credit :0
write in **credit : 5**
result : debit : 10, **credit :0** --> strange

normal behavhior :
result : debit : 0, credit :1

You need to use two on_change

```python
@api.onchange('debit')
def _onchange_debit(self) #use private function
    self.ensure_one()
    if self.debit '!=' 0:
          self.credit = 0

@api.onchange('credit')
def _onchange_credit(self) #use private function
    self.ensure_one()
    if self.credit '!=' 0:
          self.debit = 0
```

@qdp-odoo

Reply:
Yup, this is *exactly* what i had before the force push

Reply:
:-D

Reply:
Ok, ok


and _now_ I understand why we'd need 2 onchanges. My apologize @william-andre you can force-pushback ^^

Reply:
@qdp-odoo it is possible with 1 onchange, but you need to use `self._origin` to know witch fields change.